cypress/e2e/read-receipts/missing-referents.spec.ts

```diff
@@ -0,0 +1,163 @@
+/*
+Copyright 2023 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+ * # High Level Read Receipt Tests
```

Comment:
also, presumably only `high-level.spec.ts` are "High Level" tests?

Reply:
I still consider all of these high level. I avoided renaming the "high-level" file as "misc" because I didn't like that name and it made it easer for me to preserve my later PRs.

I am kind of hoping that when we flesh out more tests inside that file they will earn their own separate files with good names.